docs/about/overview/encryption.mdx

```diff
@@ -7,103 +7,112 @@ sidebar_position: 3
+\*On the Quantum-Resistance of AES256, see:
+- [this stackoverflow question, containing a bit of debate and two good answers](https://crypto.stackexchange.com/q/6712)
 - [NIST's Post-Quantum FAQ](https://csrc.nist.gov/projects/post-quantum-cryptography/faqs) section « To protect against the threat of quantum computers, should we double the key length for AES now? (added 11/18/18) »
 
 **Recommendations** for users using *private channels*:
-- **Do not configure private channels on unattended nodes**, nodes will relay meshtastic traffic even if they are not able to decrypt it.
-  Your unattended routers should not have the private PSK configured as it is easy to gain physical access and extract the channel key.
-- Keep in mind, everything sent on a channel can be stored and decrypted later by anyone who gain access to the key even if you locally delete the messages.
+- **Do not configure private channels on unattended nodes**. Nodes will relay Meshtastic traffic even if they are not able to decrypt it. Your unattended routers should not have the private PSK configured as it is easy to gain physical access and extract the channel key.
+- Keep in mind that everything sent on a channel can be stored and decrypted later by anyone who gains access to the key, even if you locally delete the messages.
 - Change your Channel Keys from time to time.
 
 ### Integrity
 
 Integrity means ensuring messages cannot be modified without the key.
 
-Meshtastic does not check messages have not been tempered with, see [`#4030`](https://github.com/meshtastic/firmware/issues/4030) for details.
+Meshtastic does not check messages have not been tampered with, see [`#4030`](https://github.com/meshtastic/firmware/issues/4030) for details.
 
 **Recommendations**:
-- Keep in mind, by applying a known plaintext attack anyone *could* send messages on a channel even with the channel key is private and secure.
-- For developers of third party applications integrating meshtastic, include a cryptographic MAC over the message content and PSK,
-  we are also considering an [AEAD secured channel mode which would provide this natively](https://github.com/meshtastic/firmware/issues/4030).
+- Keep in mind that by applying a known plaintext attack, anyone *could* send messages on a channel even if the channel key is private and secure.
+- For developers of third-party applications integrating Meshtastic, include a cryptographic MAC over the message content and PSK. We are also considering an [AEAD secured channel mode which would provide this natively](https://github.com/meshtastic/firmware/issues/4030).
 
 ### Authentication
 
-Authentication means nodes say who they are on the network, meshtastic does not implement this so it is trivial to impersonate anyone else if you have access to the channel key.
+Authentication means nodes say who they are on the network. Meshtastic does not implement this, so it is trivial to impersonate anyone else if you have access to the channel key.
```

Comment:
This sentences is not wrong, but it is not quite correct either.

This only applies to group chats. In DMs the UI tries to be scary when a node change it's identity.